documentation/01_tutorial/04-3-groundwork.html.md

```diff
@@ -109,7 +109,8 @@ We're going to be extending the `FlxSprite` class to create our `Player` class.
 	makeGraphic(16, 16, FlxColor.BLUE);
 	```
 	
-	(Make sure that an import is created for `FlxColor`, and if it isn't, add it)
+	(`flixel.util.FlxColor`)
```

Comment:
Elsewhere these hints seem to be prefixed with `import`, for instance line 60 on this page.